StayFit/RecommendedWorkoutViewController.m

```diff
@@ -0,0 +1,167 @@
+               if (error != nil) {
+                   NSLog(@"%@", [error localizedDescription]);
+               }
+               else {
+                   self.workoutArray = [NSJSONSerialization JSONObjectWithData:data options:NSJSONReadingMutableContainers error:nil];
+                  NSLog(@"%@", self.workoutArray);
+                    [self.tableView reloadData];
+                   }
+        }];
+        [task resume];
+    }
+    else if([currentUser[@"fitnessLevel"]  isEqual: @"Advanced"]){
+        
+        // creates NSURL object
+        NSURL *url = [NSURL URLWithString:@"https://api.api-ninjas.com/v1/exercises?difficulty=advanced"];
+        
+        // creates NSURLMutableRequest with the NSURL
+        NSMutableURLRequest *request = [NSMutableURLRequest requestWithURL: url];
+        
+        // configures the NSURLMutable Request with method and headers
+        [request setHTTPMethod:@"GET"];
+        [request setValue:@"Aq8IKvomBOkXPQELcAKs7Q==kffUR94hO4SaCim1" forHTTPHeaderField:@"X-Api-Key "];
+        [request setValue:@"application/json" forHTTPHeaderField:@"Content-Type"];
+        
+        // creates session
+        NSURLSession *session = [NSURLSession sessionWithConfiguration:[NSURLSessionConfiguration defaultSessionConfiguration] delegate:nil delegateQueue:[NSOperationQueue mainQueue]];
+        
+        //creates  session task
+        NSURLSessionDataTask *task = [session dataTaskWithRequest:request completionHandler:^(NSData *data, NSURLResponse *response, NSError *error) {
+               if (error != nil) {
+                   NSLog(@"%@", [error localizedDescription]);
+               }
+               else {
+                   self.workoutArray = [NSJSONSerialization JSONObjectWithData:data options:NSJSONReadingMutableContainers error:nil];
+                  NSLog(@"%@", self.workoutArray);
+                    [self.tableView reloadData];
+                   }
+        }];
+        [task resume];
+    }
+}
+
+- (UITableViewCell *)tableView:(UITableView *)tableView cellForRowAtIndexPath:(NSIndexPath *)indexPath{
+    WorkoutCell *cell = [tableView dequeueReusableCellWithIdentifier:@"workout"];
+    NSDictionary *workout = self.workoutArray[indexPath.section];
+    cell.layer.cornerRadius = 7.0f;
+    cell.layer.masksToBounds = YES;
+    cell.workoutName.text = workout[@"name"];
+    cell.muscle.text = [NSString stringWithFormat:@"%@%@", @"Muscle targeted: ", workout[@"muscle"]];
+    cell.type.text = [NSString stringWithFormat:@"%@%@", @"Type of workout: ", workout[@"type"]];
+    cell.instructions.text = workout[@"instructions"];
+    cell.layer.masksToBounds = false;
+    cell.layer.shadowOpacity = 0.23;
+    cell.layer.shadowRadius = 4;
+    return cell;
+}
+
+- (CGFloat)tableView:(UITableView *)tableView heightForRowAtIndexPath:(NSIndexPath *)indexPath
+{
+  return 120;
```

Comment:
can you specify what the number 120 means? Ideally we specify a constant to make it easier for developers to understand in the future